src/scss/core/_grid.scss

```diff
@@ -1,12 +1,20 @@
-@media (min-width: map-get($grid-breakpoints, lg)) {
-	.row {
-		margin-right: -$grid-gutter-width-lg / 2;
-		margin-left: -$grid-gutter-width-lg / 2;
-	}
+.row {
+  margin-right: -$grid-gutter-width / 2;
+  margin-left: -$grid-gutter-width / 2;
```

Comment:
as we discussed with Lusia during the call, I would rather prefer to have the additional 7.5px as padding of the container on mobile. Since it's an exception we should also incapsulate it into a media-query exclusive for the all breakpoints up to lg

Reply:
i think default value of variable $grid-gutter-width 30px and i override variable for as first mobile is $grid-gutter-width: 15 so i think its correct which padding left and right value would come 7.5px, do you mean padding value for mobile is 15px left and 15px right?